### PR TITLE
Batch model adds and trigger a single action

### PR DESCRIPTION
**Pull request by Morantron**
Improve performance by batching model adds.

### CODE REVIEW

File: src/model-add-batcher.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    this.flushAfter();
+    this.models.push(model);
+  }
+
+  flushAfter() {
+    if (this.models.length > 0) {
+      return;
+    }
+
+    defer(() => this.flush());
```
Comment (somebody32) [inline src/model-add-batcher.js:23]: do we really need to bring that dependency?

File: src/ear-fabric.js [hunk 2]
```diff
@@ -78,10 +79,11 @@ function createHandlersWithActions(rawActions, dispatch) {
  */
 export default function(collection, rawActions, dispatch) {
   const handlers = createHandlersWithActions(rawActions, dispatch);
+  const modelAddBatcher = new ModelAddBatcher({handle: handlers.handleAdd});
 
   handlers.initialSync(collection.models || collection);
 
-  collection.on('add', handlers.handleAdd);
+  collection.on('add', (model) => modelAddBatcher.add(model));
```
Comment (somebody32) [inline src/ear-fabric.js:86]: should all events behave like that?

Comment (Morantron) [reply src/ear-fabric.js:86]: I think we can do that at a later release :+1:

File: src/model-add-batcher.js (hunk 1 above)
Comment (Morantron) [reply src/model-add-batcher.js:23]: IMO microdependencies like this are good :+1:

**Pull request merged**